Formula/never.rb

```diff
@@ -20,10 +20,10 @@ class Never < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "6075cc666f6d3d9033692d0ccb7e802411af1ec7d514dbe8bb83ab1001bf3f1c"
   end
 
+  depends_on "bison" => :build
   depends_on "cmake" => :build
+  depends_on "flex" => :build
```

Comment:
why change `flex`

Reply:
it is just a build dependency, I feel like we use both the latest versions of bison and flex. what do you think?

Reply:
Let's not add dependencies we don't need; this leads to extra CI time for those formulae.

Reply:
ok